lib/doorkeeper/orm/active_record/application.rb

```diff
@@ -12,6 +12,14 @@ class Application < ActiveRecord::Base
     validates :uid, uniqueness: true
     validates :redirect_uri, redirect_uri: true
 
+    validates_each :scopes, if: :enforce_scopes? do |record, attr, value|
+      valstr = value.to_s
+      unless valstr.blank? ||
```

Comment:
Lets rewrite it to `if` - `unless` with more than one condition is hard to read and maintain.